TestProbitCG: 8 failing tests (Python 3.4 / Ubuntu 12.04)

I am attempting to install statsmodels master on Python 3.4 for the [Snake Charmer](https://github.com/andrewclegg/snake-charmer/) project. Several tests are failing or throwing errors.

Python 3.4
Ubuntu 12.04 64 bit (virtual)
NumPy 1.8.1
SciPy 0.13.1

statsmodels revision 3adaa1a0738131bd2441eb45b1c090a4777d7068
Snake Charmer revision https://github.com/andrewclegg/snake-charmer/commit/72734deb5b3a5d9cf4848aeb6b19a4661a85efe7

To reproduce: install Snake Charmer, spin up a VM as directed in the README, log in using `vagrant ssh`, test as normal.

```
======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_bse
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 85, in test_bse
    assert_almost_equal(self.res1.bse, self.res2.bse, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 50.0%)
 x: array([ 0.69414806,  0.08390496,  0.59514105,  2.54327138])
 y: array([ 0.69388252,  0.08389026,  0.59503792,  2.5424725 ])

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_conf_int
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 56, in test_conf_int
    assert_allclose(self.res1.conf_int(), self.res2.conf_int, rtol=8e-5)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 1183, in assert_allclose
    verbose=verbose, header=header)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Not equal to tolerance rtol=8e-05, atol=0

(mismatch 100.0%)
 x: array([[  0.26725425,   2.98826463],
       [ -0.1127225 ,   0.21617888],
       [  0.25981741,   2.59272747],
       [-12.44303176,  -2.47359115]])
 y: array([[  0.2658255,   2.985795 ],
       [ -0.1126929,   0.2161508],
       [  0.2600795,   2.592585 ],
       [-12.43547  ,  -2.469166 ]])

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_params
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 53, in test_params
    assert_almost_equal(self.res1.params, self.res2.params, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 50.0%)
 x: array([ 1.62775944,  0.05172819,  1.42627244, -7.45831145])
 y: array([ 1.62581025,  0.05172895,  1.42633237, -7.45232042])

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_predict
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 99, in test_predict
    self.res2.phat, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 43.75%)
 x: array([ 0.01813411,  0.05303997,  0.19003049,  0.01855665,  0.55528215,
        0.02720612,  0.018475  ,  0.04453257,  0.1087887 ,  0.66371484,
        0.01606698,  0.19368361,  0.32366954,  0.1952776 ,  0.35666492,...
 y: array([ 0.0181707,  0.0530805,  0.1899263,  0.0185707,  0.5545748,
        0.0272331,  0.0185033,  0.0445714,  0.1088081,  0.6631207,
        0.0161024,  0.1935566,  0.3233282,  0.1951826,  0.3563406,...

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_predict_xb
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 104, in test_predict_xb
    self.res2.yhat, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 87.5%)
 x: array([-2.09390756, -1.61606651, -0.87778395, -2.08451562,  0.13901828,
       -1.92354024, -2.08631619, -1.70034989, -1.23299563,  0.42262303,
       -2.14274035, -0.86440177, -0.45746191, -0.85861095, -0.36738772,...
 y: array([-2.093086  , -1.61569178, -0.87816805, -2.08420706,  0.13722852,
       -1.92311108, -2.08569193, -1.69993722, -1.23289168,  0.42099541,
       -2.14186025, -0.86486465, -0.45841211, -0.85895526, -0.36825761,...

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_resid_dev
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 124, in test_resid_dev
    DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 68.75%)
 x: array([-0.19131412, -0.3301466 , -0.64924367, -0.19355092,  1.08469251,
       -0.23487468, -0.19312064, -0.301843  , -0.47994526,  0.90543103,
       -0.17998583, -0.6561693 , -0.88439072,  1.80739213, -0.93924391,...
 y: array([-0.191509 , -0.3302762, -0.6490455, -0.1936247,  1.085867 ,
       -0.2349926, -0.1932698, -0.3019776, -0.4799906,  0.9064196,
       -0.1801855, -0.6559291, -0.8838201,  1.807661 , -0.9387071,...

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_resid_generalized
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 128, in test_resid_generalized
    self.res2.resid_generalized, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 56.25%)
 x: array([-0.04537189, -0.11414615, -0.33506451, -0.04629088,  0.71154071,
       -0.06448265, -0.04611355, -0.09837399, -0.20931732,  0.54972289,
       -0.0408271 , -0.34052911, -0.53126158,  1.41310335, -0.57964708,...
 y: array([-0.045452, -0.11422 , -0.334908, -0.046321,  0.712624, -0.064538,
       -0.046175, -0.098447, -0.209349,  0.550593, -0.040906, -0.340339,
       -0.530763,  1.413373, -0.57917 , -0.053593, -0.100556, -0.071855,...

======================================================================
FAIL: statsmodels.discrete.tests.test_discrete.TestProbitCG.test_zstat
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/local/lib/python3.4/dist-packages/statsmodels/discrete/tests/test_discrete.py", line 59, in test_zstat
    assert_almost_equal(self.res1.tvalues, self.res2.z, DECIMAL_4)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 454, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 811, in assert_array_almost_equal
    header=('Arrays are not almost equal to %d decimals' % decimal))
  File "/usr/local/lib/python3.4/dist-packages/numpy/testing/utils.py", line 644, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to 4 decimals

(mismatch 75.0%)
 x: array([ 2.34497443,  0.61650933,  2.39652841, -2.93256611])
 y: array([ 2.34306269,  0.61662638,  2.39704449, -2.93113118])

----------------------------------------------------------------------
```

--- a/statsmodels/discrete/tests/test_discrete.py
+++ b/statsmodels/discrete/tests/test_discrete.py
@@ -378,7 +378,7 @@ class TestProbitCG(CheckBinaryResults):
                                                      maxiter=500, gtol=1e-08,
                                                      disp=0)
 
-        assert_array_less(cls.res1.mle_retvals['fcalls'], 70)
+        assert_array_less(cls.res1.mle_retvals['fcalls'], 100)
 
 
 class TestProbitNCG(CheckBinaryResults):

